documentation/src/main/asciidoc/topics/proc_cli_shutdown.adoc

```diff
@@ -1,17 +1,28 @@
 [id='server_shutdown-{context}']
 = Shutting Down {brandname} Server
-Gracefully shut down running servers to passivate all entries to disk and persist state.
+Stop individually running servers or bring down clusters gracefully.
+
+[IMPORTANT]
+====
+When you stop server instances, you should always wait for rebalancing operations to complete to avoid losing data.
```

Comment:
there are 2 important things here: 
* Only caches with persistence configure will keep their state. Caches without persistence will lose all data
* If `shutdown server...` is used, the servers must be stopped one by one, wait for rebalance and the restart *must* happen in the inverse order. If you miss one of these steps, data loss may occur.

Reply:
@pruivo and @wfink Should there be an exception if more than one server is specified with `shutdown server`?

`shutdown server server1 server2` does not wait for rebalancing so can lead to data loss. We should recommend against this usage and disallow it if possible imo.

Reply:
As far I understand it you can do it similar to killing nodes
So server shutdown is not graceful, but shutdown cluster is.

We might add a "--force" to allow more than one server,
but at this point we don't know the numOwner configuration for every cache, so it is difficult to decide whether 2 or more are allowed.
So I would leave it to the user to know it is loosing data, or have some automatic to stop one after the other.

So we document that shutdown servers migth lose data even if not shutting down the complete cluster.
And if no persistence all data is lost if all nodes are down

@pruivo what do you think?

Reply:
well, the user is shut down for good (i.e. no intention to restart), `shutdow server` may be useful. I would leave it as is.